File: .github/src/model/Money.py
from model.common.WalletObject import WalletObject


class Money(WalletObject):
    def __init__(self, id, amount):
        super().__init__(id)
        self._amount = amount

    def get_amount(self):
        return self._amount

    def set_amount(self, amount):
        self._amount = amount

    def __str__(self):
        return f"Money{{amount={self._amount}}}"

File: model/Money.js
import WalletObject from "./common/wallet-object";


class Money extends WalletObject {
    constructor(id, amount) {
        super(id);
        this._amount = amount;
    }

    getAmount() {
        return this._amount;
    }

    setAmount(amount) {
        this._amount = amount;
    }

    toString() {
        return `Money{amount=${this._amount}}`;
    }
}

export default Money;